bluetooth/src/main/java/me/aflak/bluetooth/Bluetooth.java

```diff
@@ -288,9 +300,18 @@ public void connectToDeviceWithPortTrick(BluetoothDevice device){
     }
 
     /**
-     * Disconnect from bluetooth device.
+     * Disconnect from all bluetooth devices.
      */
     public void disconnect() {
+        for (ReceiveThread receiveThread: receiveThreads) {
```

Comment:
We want to able to control each socket separately. We should not disconnect every socket.

Reply:
Or you should rename the method `disconnectAll()`